drivers/ethernet/Kconfig

```diff
@@ -99,3 +99,12 @@ config ETH_INIT_PRIORITY
 	  Do not mess with it unless you know what you are doing.
 	  Note that the priority needs to be lower than the net stack
 	  so that it can start before the networking sub-system.
+
+config ETH_NET_IF_NO_AUTO_START
+	bool "Disable Ethernet interface auto-start"
+	help
```

Comment:
Should add a
```
depends on NET_L2_ETHERNET
```
or at least a
```
depends on NETWORKING
```
Don't want to "polute" the Kconfig, when Ethernet is not activated.